src/sass/partials/global/_mixins.scss

```diff
@@ -78,6 +78,17 @@
       box-shadow: inset 0 0 0 setting-get('input border width') / 1.5 setting-get('white');
     }
   }
+
+  &:focus {
+    outline: 0 none;
+
+    + label {
+
+      &::before {
+        outline: 4px solid setting-get('primary accent color');
```

Comment:
Is this the same outline we have throughout Punchcard? If so, we should probably turn it in to a variable to better reuse it